t/Statements/Expressions/Value/NonLiteral/Variable.t

```diff
@@ -91,6 +91,7 @@ parses('$^X');
 parses('@+');
 parses('@-');
 parses('@_');
+parses('$#_');
```

Comment:
Please add tests for `$#+` and `$#-` as well.

Reply:
Just adding tests doesn't do any good
```
t/Statements/Expressions/Value/NonLiteral/Variable.t ............... 68/? #

#   Failed test ''$#+': parsed unambiguously'
#   at t/Statements/Expressions/Value/NonLiteral/Variable.t line 92.
#          got: '0'
#     expected: '1'
#

#   Failed test ''$#-': parsed unambiguously'
#   at t/Statements/Expressions/Value/NonLiteral/Variable.t line 94.
#          got: '0'
#     expected: '1'
t/Statements/Expressions/Value/NonLiteral/Variable.t ............... 87/? # Looks like you failed 2 tests of 96.
t/Statements/Expressions/Value/NonLiteral/Variable.t ............... Dubious, test returned 2 (wstat 512, 0x200)
Failed 2/96 subtests
```
That's surprising, as '-'/'+' are in GlobalVariables and should be parsed.